core/transientstore/store.go

```diff
@@ -98,6 +114,173 @@ func (provider *storeProvider) Close() {
 	provider.dbProvider.Close()
 }
 
+// DeleteStore deletes the TransientStoreProvider for a given ledger.
```

Comment:
```suggestion
// DeleteStore deletes the TransientStore for a given ledger.
```
TransientStoreProvider is a single instance shared across channels.